tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -25,17 +25,12 @@
 #include <utility>
 #include <valarray>
 #include <vector>
-
-#define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)
-
-namespace ranges = std::ranges;
+//

```

Comment:
Ha there it is an unnecessary `//``
```suggestion

```

Reply:
This keeps clang-format from sorting `range_algorithm_support.hpp` in with the other headers - it's fairly common to in all of the ranges algorithm tests. I like include all product headers before any test headers to avoid inadvertent dependencies.

Reply:
(There are a few other patterns where we use `//` with nothing after it to force clang-format's hand.)

Reply:
Ah your rules remove the empty line and then sort. Thanks for the explanation.

Reply:
@CaseyCarter Idea: Should we consistently name everything in `tests/std/include` as `test_meow.hpp`, then teach `clang-format` to sort `test_meow.hpp` at the end?

Reply:
I like this general idea, but it should be a separate change: my investigation is producing 1.3k additional lines of diff.

Reply:
Be careful what you ask for: #902.